app/views/browse/service-instance.html

```diff
@@ -0,0 +1,102 @@
+                <ul class="dropdown-menu dropdown-menu-right actions action-button">
+                  <li ng-if="{resource: 'instances', group: 'servicecatalog.k8s.io'} | canI : 'update'">
+                    <a ng-href="{{serviceInstance | editYamlURL}}" role="button">Edit YAML</a>
+                  </li>
+                  <li ng-if="{resource: 'instances', group: 'servicecatalog.k8s.io'} | canI : 'delete'">
+                    <delete-link
+                      kind="Instance"
+                      resource-name="{{serviceInstance.metadata.name}}"
+                      project-name="{{serviceInstance.metadata.namespace}}"
+                      alerts="alerts">
+                    </delete-link>
+                  </li>
+                </ul>
+              </div>
+              {{serviceInstance | serviceInstanceDisplayName:serviceClasses}}
+              <small class="meta">created <span am-time-ago="serviceInstance.metadata.creationTimestamp"></span></small>
+            </h1>
+            <labels labels="serviceInstance.metadata.labels" clickable="true" kind="instance" project-name="{{serviceInstance.metadata.namespace}}" limit="3"></labels>
+          </div>
+        </div>
+      </div><!-- /middle-header-->
+      <div class="middle-content" persist-tab-state>
+        <div class="container-fluid">
+          <div class="row" ng-if="serviceInstance">
+            <div class="col-md-12">
+              <uib-tabset>
+                <uib-tab active="selectedTab.details">
+                  <uib-tab-heading>Details</uib-tab-heading>
+                  <div class="resource-details">
+                    <div class="row">
+                      <div class="col-lg-6">
+                        <h3>Plan</h3>
+                        <p>
+                          {{plan.description}}
+                        </p>
+                        <dl class="dl-horizontal left">
+                          <dt ng-if-start="serviceClass.description">Description:</dt>
+                          <dd ng-if-end>
+                            <p class="pre-wrap" ng-bind-html="serviceClass.description | linky"></p>
+                            <p class="pre-wrap" ng-bind-html="serviceClass.externalMetadata.longDescription | linky"></p>
+                          </dd>
+                          <dt>Status:</dt>
+                          <dd>
+                            <status-icon status="serviceInstance | serviceInstanceStatus" disable-animation></status-icon>
+                            <span flex>{{serviceInstance | serviceInstanceStatus | sentenceCase}}</span>
+                          </dd>
+                          <dt ng-if-start="serviceInstance | serviceInstanceMessage">Status Reason:</dt>
+                          <dd ng-if-end>
+                            {{serviceInstance | serviceInstanceMessage}}
+                          </dd>
```

Comment:
We've used a somewhat different layout for status on the route page and the build config page